src/components/Heading/Heading.tsx

```diff
@@ -92,12 +93,21 @@ export const Heading = forwardRef(
       }
     }
 
+    let variantComputed: string | undefined = variant;
+    if (variant === 'neutral-subtle') {
+      variantComputed = 'neutral-subtle';
+    } else if (variant === 'neutral-medium') {
+      variantComputed = 'neutral-medium';
+    } else if (variant === 'neutral-strong') {
+      variantComputed = 'neutral-strong';
+    }
```

Comment:
can we remove this and just use `variant`?

Reply:
Oh yeah, thanks for pointing that out!